2_process/src/create_lake_task_plan.R

```diff
@@ -8,37 +8,56 @@ create_lake_task_makefile <- function(makefile, task_plan, remake_file, final_ta
 }
 
 create_lake_tasks <- function(task_df_fn, log_folder){
-  
-  # prepare a data.frame with one row per task
-  tasks <- readRDS(task_df_fn)[1:3,] %>% 
-    rename(task_id = nhdhr, task_filename = filename) %>% 
-    mutate(task_name = task_id) %>% 
-    # Remove the lakes that didn't merge data
-    filter(nchar(task_filename) != 0) 
+
+  tasks <- tibble(task_filepath = names(yaml::yaml.load_file(task_df_fn))) %>% 
```

Comment:
OK, this is what I couldn't figure out!

Reply:
So if you wanted to do pdgl, would you make a completely different task table? I was thinking that we could just add a step here, but was struggling with making it generic enough.